content/blog/2024/10/kubewarden-1-18-release-slsa-level-3.md

```diff
@@ -0,0 +1,122 @@
+  [`rancher/kuberlr-kubectl`](https://github.com/rancher/kuberlr-kubectl), which
+  comes with provenance, and provides cached kubectl commands useful for airgap
+  installs.
+
+We have decided to implement provenance for policies on a later date.
+Currently, policies can be either out of band OCI artifacts (usually from
+artifacthub.io), or just a YAML definition (e.g: CEL policies using
+[cel-policy](https://artifacthub.io/packages/kubewarden/cel-policy/cel-policy))
+or Kyverno DSL policies using [kyverno-dsl
+-policy](https://github.com/kubewarden/kyverno-dsl-policy). In addition, all
+policies are sandboxed thanks to WebAssembly.
+
+### Verifying the attestations
+
+For verifying the attestations, have a look at our
+[docs](https://docs.kubewarden.io/tutorials/verifying-kubewarden). As usual
+we sign our attestations via Sigstore with keyless workflow.
+
+The UX for verifying provenance attestations is still somewhat green across
+this space. For binaries such as `kwctl` is as easy as:
+
+```console
+$ gh attestation verify kwctl-linux-x86_64 --repo kubewarden/kwctl
+Loaded digest sha256:64e0e28eef99f69a1e86f2677bb9c506c3fafe6d743c354b8a4b1c1b5ac90176 for file://kwctl-linux-x86_64
+Loaded 2 attestations from GitHub API
+✓ Verification succeeded!
+
+sha256:64e0e28eef99f69a1e86f2677bb9c506c3fafe6d743c354b8a4b1c1b5ac90176 was attested by:
+REPO              PREDICATE_TYPE                  WORKFLOW
+kubewarden/kwctl  https://slsa.dev/provenance/v1  .github/workflows/build.yml@refs/tags/v1.18.0-rc1
+kubewarden/kwctl  https://slsa.dev/provenance/v1  .github/workflows/build.yml@refs/heads/main
+```
+
+For container images it may mean first finding the attestation manifest index
+layer, and then the attestation layer itself. If in doubt, read each project's
+developer docs in their respective GitHub repositories.
+
+## shortNames for our CRDs
+
+Kubernetes supports short names for CRDs, and after the new PolicyGroups
+feature, we think it's the perfect moment to include them. Short names simplify
+day-to-day operations. For example, to list ClusterAdmissionPolicies:
+
+```
+kubectl get cap
+NAME                        POLICY SERVER   MUTATING   BACKGROUNDAUDIT   MODE      OBSERVED MODE   STATUS   AGE
+do-not-run-as-root          default         true       true              monitor   monitor         active   3d
+do-not-share-host-paths     default         false      true              monitor   monitor         active   3d
+...
+```
+
+That's a lot better, right?
+
+Here's our shortened resource types:
+
+| Resource                     | shortName |
+| ---------------------------- | --------- |
+| AdmissionPolicies            | **ap**    |
+| ClusterAdmissionPolicies     | **cap**   |
+| AdmissionPolicyGroups        | **apg**   |
+| ClusterAdmissionPolicyGroups | **capg**  |
+| PolicyServers                | **ps**    |
+
+## Future deprecation of CRDs version `v1alpha2`
+
+In future releases, we intend to deprecate our CRDs version `v1alpha2` in
```

Comment:
Actually, we [deprecated](https://github.com/kubewarden/kubewarden-controller/pull/896) the `v1alpha2` in the `v1.18.0` release. Together with the shortnames  ;)

Reply:
agh, indeed :D ! I will change the wording to reflect it goes into this release.

Reply:
Done https://github.com/kubewarden/kubewarden.io/pull/275/commits/5c57da712a6c20c82027fbb031ca31c20ad2b11c.

Reply:
Thanks!